src/react-queries/queries/swapper.ts

```diff
@@ -0,0 +1,33 @@
+import { createQueryKeys } from '@lukemorales/query-key-factory'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { type HDWallet, supportsETH } from '@shapeshiftoss/hdwallet-core'
+import type { GetEvmTradeQuoteInput } from '@shapeshiftoss/swapper'
+import { arbitrumBridgeApi } from 'lib/swapper/swappers/ArbitrumBridgeSwapper/endpoints'
+import { getTradeQuote } from 'lib/swapper/swappers/ArbitrumBridgeSwapper/getTradeQuote/getTradeQuote'
+
+// Only used for arbitrum bridge swapper use outside of swapper for now
```

Comment:
Wondering if we'll ever escape-hatch swapper somewhere else, IMO not much sense to abstract this some more and make it swapper-agnostic

Reply:
See thread: https://discord.com/channels/554694662431178782/1248083320781144085
Final word is we probably won't so not much sense to abstract things to be more swapper-y.